olm/update-nightly-olm-files.sh

```diff
@@ -33,13 +33,26 @@ do
     echo "   - Updating new package version with roles defined in: ${role}"
     cp "$role" generated/current-role.yaml
     operator-sdk olm-catalog gen-csv --csv-version "${newNightlyPackageVersion}" --from-version="${lastPackageVersion}" 2>&1 | sed -e 's/^/      /'
+
+    containerImage=$(sed -n 's|^ *image: *\([^ ]*/che-operator:[^ ]*\) *|\1|p' "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml")
+    createdAt=$(date -u +%FT%TZ)
+    echo "   - Updating new package version fields:"
+    echo "       - containerImage => ${containerImage}" 
+    echo "       - createdAt => ${createdAt}" 
+    sed \
+    -e "s|containerImage:.*$|containerImage: ${containerImage}|" \
+    -e "s/createdAt:.*$/createdAt: \"${createdAt}\"/" \
+    "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml" \
+    > "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml.new"
+    
+    mv "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml.new" \
+    "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml"
+
   done
   echo "   - Copying the CRD file"
   cp "${packageFolderPath}/${lastPackageVersion}/eclipse-che-preview-${platform}.crd.yaml" "${packageFolderPath}/${newNightlyPackageVersion}/eclipse-che-preview-${platform}.crd.yaml"
   echo "   - Updating the 'nightly' channel with new version in the package descriptor: ${packageFilePath}"
-  echo "     (the previous one is saved with the .old suffix)"
   sed -e "s/${lastPackageVersion}/${newNightlyPackageVersion}/" "${packageFilePath}" > "${packageFilePath}.new"
```

Comment:
if you use `-i` you can apply the sed transform in-place to the file, without having to redirect to .new then move .new to the actual filename.

`sed -i -e "s..." filename`

Reply:
see the answer [here](https://github.com/eclipse/che-operator/pull/50/files#r303958738)